docs/maui/views/camera-view.md

```diff
@@ -155,6 +155,19 @@ Tizen is not currently supported.
 
 The `CameraView` can be added to a .NET MAUI application in the following way.
 
+### Request permissions
+
+Developers must manually request Permissions.Camera and/or Permissions.Microphone:
```

Comment:
Like the folder picker perhaps we can explain the reasoning inline to make it easier for the developer to choose?